components/scripts/gradle/gradle-init-scripts/configure-gradle-enterprise.gradle

```diff
@@ -72,7 +72,6 @@ def ccudPluginVersion = getInputParam('com.gradle.enterprise.build-validation.cc
 def expDir = getInputParam('com.gradle.enterprise.build-validation.expDir')
 def expId = getInputParam('com.gradle.enterprise.build-validation.expId')
 def runId = getInputParam('com.gradle.enterprise.build-validation.runId')
-def runNum = getInputParam('com.gradle.enterprise.build-validation.runNum')
```

Comment:
Add a comment above this block (and the one above) explaining that these variables do not change between the two invocations of the build; thus, they do not need to be tracked as inputs to the configuration model.